crates/re_types/definitions/rerun/datatypes/angle.fbs

```diff
@@ -12,9 +12,11 @@ namespace rerun.datatypes;
 union Angle (
   "attr.rust.derive": "Copy, PartialEq"
 ) {
+  /// \rs Angle in radians. One turn is equal to 2π (or τ) radians.
   /// \py 3D rotation angle in radians. Only one of `degrees` or `radians` should be set.
```

Comment:
the one turn clarifaction is nice, can also go on python.
The fact that python says "3d rotation angle" is also a kinda waiting to be wrong - surely we gonna use the same thing both in 2D!